preprod.healdata.org/metadata/aggregate_config.json

```diff
@@ -194,7 +194,7 @@
         },
         "ICPSR": {
```

Comment:
let's also remove the section about `SAMHDA` from this file https://github.com/uc-cdis/cdis-manifest/pull/7244/files#diff-67d6eecf0e0414ddd5ce26d56cf1ca73e4ad96d94b2b055ab83e8d99930dcaf4L94